src/js/components/hyperdrive-size/index.js

```diff
@@ -30,8 +30,10 @@ HyperdriveSize.prototype._updateSize = function (state) {
     var s = state.archiveReducer
     if (s.archive && s.archive.content && s.archive.content.bytes) {
       this._size = prettyBytes(s.archive.content.bytes)
+      this.$el.style.display = 'block'
     } else {
       this._size = 0
+      this.$el.style.display = 'none'
```

Comment:
this can be easily fixed in another PR, can you just add a `TODO: fix` comment here instead? also, if we do any dom manipulations re: style display or block, they should always go in component _render() method, otherwise it can be confusing.